Bottom: ff4539e26e4746582d6365b9481f80cce66a2911
Top:    8badf98b22d0b32f2a07ffa9f6ae4f6e930a790b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-06 00:42:37 +0100

Refresh of add-a-hid-api-to-notify-the-gu

---

diff --git a/src/hid.h b/src/hid.h
index c120164..2266b6c 100644
--- a/src/hid.h
+++ b/src/hid.h
@@ -579,7 +579,7 @@ typedef enum
     void (*notify_save_pcb) (const char *filename, bool done);
 
     /* Notification to the GUI that the PCB file has been renamed. */
-    void (*notify_filename_changed) (void)
+    void (*notify_filename_changed) (void);
   };
 
 /* Call this as soon as possible from main().  No other HID calls are
